addons/mail/tools/followers.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+
+_UNFOLLOW_REGEX = re.compile(r'unfollow_placeholder_start(.*)unfollow_placeholder_end', re.DOTALL)
```

Comment:
Actually it was to not make mail_thread dependents of mail_mail or vice-versa (because it is used in both). What do you think ? (we end up easily with cyclic dependencies when importing a model from another)

Reply:
if you do something like this, it should work I think
```
from odoo.mail.... import UNFOLLOW_REGEX
```

but in the end maybe we don't even need this regex if we do the same logic as `_replace_local_links` (see others comments)